tools/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/client9/misspell v0.3.4
 	github.com/golangci/golangci-lint v1.32.2
 	github.com/katbyte/terrafmt v0.2.1-0.20200913185704-5ff4421407b4
+	github.com/pavius/impi v0.0.3 // indirect
```

Comment:
Whether this came in by accident or from the merge, it seems worth rebasing and taking this out.

I can help you with how to rebase if you want a second pair of eyes!

Reply:
I think all of go.mod files were mistakes.  At the time it was required because I needed the latest aws-go lib.  I attempted to revert it but I don't know if I did it correctly.  If it isn't I would welcome some help.

Reply:
Hmm, you've built a little merge-history puzzle!

Using an alias [I have defined](https://gist.github.com/kevincantu/aa5b6875a3fdebf71f1241b56f4f6375):
```
% git la -8          
*   6bd6a2037 <Mike> (HEAD, fivehole/f-lb-target-group-protocol-version) Merge branch 'f-lb-target-group-protocol-version' of https://github.com/fivehole68/terraform-provider-aws into f-lb-target-group-protocol-version [ae448c373 b5065c049]
|\  
| * b5065c049 <Mike> fix terrafmt errors [e93c7ae6b]
| *   e93c7ae6b <fivehole68> Merge branch 'master' into f-lb-target-group-protocol-version [c76539c8c f66074baa]
| |\  
| * | c76539c8c <Mike> Remove comment [484151258]
| * | 484151258 <Mike> Add support for protocol version to target groups [b1d2e1dbc]
* | | ae448c373 <Mike> Revert go files [74d6b7c9d]
* | | 74d6b7c9d <Mike> Add support for protocol version to target groups [f66074baa]
| |/  
|/|   
* | f66074baa <Dirk Avery> Update CHANGELOG.md [2713c9186]
```

I know I mentioned rebasing, but probably the easier next step is to just merge in the latest master and overwrite a couple of those go.mod and go.sum files.

Something like this might do the trick:
```
# your branch
git checkout f-lb-target-group-protocol-version
git log --decorate --graph --oneline -10
git diff --stat hashicorp/master f-lb-target-group-protocol-version

# fetch and merge in the latest upstream
git remote -v # below I assume Hashicorp's remote is named hashicorp
git fetch hashicorp
git merge hashicorp/master

# now look at your branch
git log --decorate --graph --oneline -10
git diff --stat hashicorp/master f-lb-target-group-protocol-version

# use checkout to overwrite a couple unintentional differences
git checkout hashicorp/master tools/go.mod
git checkout hashicorp/master tools/go.sum
git commit -am "Fix dep file merge issues"
...
```

Reply:
Looks like that worked.  Thanks